examples/hotrod/services/driver/redis.go

```diff
@@ -35,27 +35,34 @@ import (
 
 // Redis is a simulator of remote Redis cache
 type Redis struct {
-	tracer opentracing.Tracer // simulate redis as a separate process
+	tracer trace.TracerProvider // simulate redis as a separate process
 	logger log.Factory
 	errorSimulator
 }
 
 func newRedis(otelExporter string, metricsFactory metrics.Factory, logger log.Factory) *Redis {
 	return &Redis{
-		tracer: tracing.Init("redis", otelExporter, metricsFactory, logger),
+		tracer: tracing.InitOTEL("redis", otelExporter, metricsFactory, logger),
 		logger: logger,
 	}
 }
 
 // FindDriverIDs finds IDs of drivers who are near the location.
 func (r *Redis) FindDriverIDs(ctx context.Context, location string) []string {
-	if span := opentracing.SpanFromContext(ctx); span != nil {
+	tracer := r.tracer.Tracer("")
+	if _, span := tracer.Start(ctx, "FindDriverIDs", trace.WithSpanKind(trace.SpanKindClient) /*...set child of option*/); span != nil {
+		span.SetAttributes(attribute.Key("param.location").String(location))
```

Comment:
let's change this a bit:
```suggestion
		span.SetAttributes(attribute.Key("param.driver.location").String(location))
```